src/create_disk.sh

```diff
@@ -53,16 +55,22 @@ do
 done
 ostree admin deploy "$ref" --sysroot rootfs --os "$os_name" $kargsargs
 
-# install bios grub
-grub2-install \
-	--target i386-pc \
-	--boot-directory rootfs/boot \
-	$disk
+if [ "$arch" == "x86_64" ]; then
+	# install bios grub
+	grub2-install \
+		--target i386-pc \
+		--boot-directory rootfs/boot \
+		$disk
+fi
+mkdir -p rootfs/boot/grub2
+
+ext="X64"
+[ "$arch" == "aarch64" ] && ext="AA64"
```

Comment:
Won't this fail and cause the script to exit on `x86_64`?

Could probably collapse these two lines into the previous `arch` conditional instead.